apis/apiextensions/fn/proto/v1alpha1/run_function.pb.go

```diff
@@ -17,7 +17,7 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.29.1
+// 	protoc-gen-go v1.28.1-devel
```

Comment:
This change probably doesn't need to be in this PR.

Reply:
reverted